src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.spring.controller;
+
+import java.util.NoSuchElementException;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.dto.mapping.DtoResponseMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final MovieSessionService movieSessionService;
+    private final DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> dtoResponseMapper;
+
+    public ShoppingCartController(ShoppingCartService shoppingCartService,
```

Comment:
```suggestion
    @Autowired
    public ShoppingCartController(ShoppingCartService shoppingCartService,
```

We need to use @Autowired  annotation above constructors everywhere or nowhere to make our code more consistent. Remove it everywhere or add it everywhere